guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -116,7 +116,8 @@ Ensure {SmartProxy} has access to `{RepoRHEL7ServerSatelliteMaintenanceProductVe
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-maintain} self-upgrade
+# subscription-manager repos --enable {RepoRHEL7ServerSatelliteMaintenanceProductVersion}
+# yum --disableplugin=foreman-protector update -y rubygem-foreman_maintain {foreman-maintain}
```

Comment:
In general we have the convention to avoid `-y` for interactive commands:
```suggestion
# yum --disableplugin=foreman-protector update rubygem-foreman_maintain {foreman-maintain}
```

Reply:
Addressed :+1: